vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -1,6 +1,7 @@
 import dateFnsFormat from 'date-fns/format';
 import dateFnsParse from 'date-fns/parse';
 import dateFnsIsValid from 'date-fns/isValid';
+import * as dateFnsLocales from 'date-fns/locale';
```

Comment:
> There is some documentation on how to

This documentation does not help us, as in our case (framework), we do not have information beforehand of the locales used by the application.

> This will essentially import all the locales

True, it will add 160kb